packages/cli/src/commands/test.js

```diff
@@ -1,20 +1,46 @@
 import execa from 'execa'
-import Listr from 'listr'
-import VerboseRenderer from 'listr-verbose-renderer'
 import terminalLink from 'terminal-link'
 
 import { getPaths } from 'src/lib'
 import c from 'src/lib/colors'
 
+const jest = require('jest')
+
+// TODO: Get from redwood.toml
+const sides = ['web', 'api']
+
+// https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/scripts/test.js#L39
+function isInGitRepository() {
+  try {
+    execa.commandSync('git rev-parse --is-inside-work-tree')
+    return true
+  } catch (e) {
+    return false
+  }
+}
+
+function isInMercurialRepository() {
+  try {
+    execa.commandSync('hg --cwd . root')
+    return true
+  } catch (e) {
+    return false
+  }
+}
+
 export const command = 'test [side..]'
 export const description = 'Run Jest tests for api and web'
 export const builder = (yargs) => {
   yargs
-    .positional('side', {
-      choices: ['api', 'web'],
-      default: ['api', 'web'],
-      description: 'Which side(s) to test',
-      type: 'array',
+    .choices('side', sides)
+    .option('watch', {
+      type: 'boolean',
+    })
+    .option('watchAll', {
+      type: 'boolean',
+    })
```

Comment:
What's the distinction between --watch and --watchAll?

Reply:
`--watch` will run only the tests that are related to files you've changed.
`--watchAll` will run ALL your tests.

Both rerun when changing a file.